src/Altinn.App.Api/Controllers/Conventions/EnumAsNumberFormatter.cs

```diff
@@ -0,0 +1,59 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Formatters;
+
+namespace Altinn.App.Api.Controllers.Conventions;
+
+internal class EnumAsNumberFormatter : SystemTextJsonOutputFormatter
```

Comment:
Do we need to implement `CanWriteResult` here? I think this formatter should only do its thing if `GetJsonSettingsName` matches the settings name?